.circleci/config.yml

```diff
@@ -10,6 +10,30 @@ jobs:
       - run:
           name: Build
           command: make image
+
+  release_images:
+    machine:
+      docker_layer_caching: true
+    working_directory: ~/codeclimate/codeclimate-phpcodesniffer
+    steps:
+      - checkout
+      - run:
+          name: Validate owner
+          command: |
+            if [ "$CIRCLE_PROJECT_USERNAME" -ne "codeclimate" ]
+            then
+              echo "Skipping release for non-codeclimate branches"
+              circleci step halt
+            fi
+      - run: make image
+      - run: echo "$GCR_JSON_KEY" | docker login -u _json_key --password-stdin us.gcr.io
+      - run:
+          name: Push image to GCR
+          command: |
+            docker tag $CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME \
+              us.gcr.io/code-climate/codeclimate-phpcodesniffer:b$CIRCLE_BUILD_NUM
+            docker push us.gcr.io/code-climate/codeclimate-phpcodesniffer:b$CIRCLE_BUILD_NUM
+
```

Comment:
Wee need the branch filtering part as well to do it only for specific branches:
https://github.com/codeclimate/shipbot/blob/3506f7ebadb6c3c3635c400e62b75dc4571cb252/lib/shipbot/commands/push_engine.rb

Reply:
Missed that thanks. Done 👍🏼